components/mixins/profile/profile-claim-process.schema.json

```diff
@@ -5,7 +5,7 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-claim-process",
```

Comment:
@mjasrotia - I made this change later in the day so you may have missed it, but claim process shouldn't be moved to profile. It should stay as EE. We just need to remove all profile data nodes from the policy data type. 

1. Update top level grouping to be "claims"
2. The mixin should only have the following fields: policy ID, start date, status, type, and currency data type (named benefitAmount- see screenshot) 
![image](https://user-images.githubusercontent.com/78930223/113803885-0fd1f680-9723-11eb-9bec-68ed320a7261.png)